Solution1/BusinessLayer/Configuration/BackgroundJobConfiguration.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+
+namespace BusinessLayer.Configuration
```

Comment:
configurations are usually in Web projects

Reply:
+